tools/metrics.py

```diff
@@ -122,8 +122,18 @@ def get_containers_healthcheck(host):
         return 1
 
     for container in data:
-        if not container['state']['Running'] or container['state']['Paused']:
-            logger.info(f'{container["name"]} is not running or paused')
+        status_ok = True
```

Comment:
It's better to move this block to the separate function - `is_container_ok` and refactor it a little bit

Reply:
done